specification/toolingservice/data-plane/1.0/toolingservice.json

```diff
@@ -0,0 +1,755 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Tooling Service Client",
+    "description": "Client that can be used to access Tooling services to run linter, static and api validations.",
+    "version": "1.0"
+  },
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "host": "<TBD>",
+  "basePath": "/api/",
+  "schemes": [
+    "https", "http"
+  ],
+  "paths": {
+    "/tools/lint": {
+      "post": {
+        "tags": [
+          "Linter"
+        ],
+        "operationId": "Tools_Linter",
+        "description": "Get Linter Result",
+        "parameters": [
+          {
+            "name":"repository",
+            "in": "body",
```

Comment:
I believe Brandon was asking in our meeting if these should be path parameters, have you evaluated that option?